ui/app/src/modules/Content/Dependencies/DependenciesDialogUI.tsx

```diff
@@ -0,0 +1,473 @@
+                emptyStateProps: {
+                  title: (
+                    <FormattedMessage
+                      id="dependenciesDialog.emptyStateMessage"
+                      defaultMessage="No Dependencies found"
+                    />
+                  ),
+                  classes: {
+                    root: classes.suspense
+                  }
+                }
+              }}
+              loadingStateProps={{
+                classes: {
+                  root: classes.suspense
+                }
+              }}
+            >
+              <DependenciesList
+                resource={resource}
+                state={state}
+                setState={setState}
+                assetsTypes={assetsTypes}
+                isEditableItem={isEditableItem}
+                handleEditorDisplay={handleEditorDisplay}
+              />
+            </SuspenseWithEmptyState>
+          </DialogBody>
+          <DialogFooter
+            classes={{
+              root: classes.dialogFooter
+            }}
+          >
+            <FormControlLabel
+              className={classes.compactViewAction}
+              control={
+                <Checkbox
+                  checked={state.compactView}
+                  onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+                    setState({ compactView: event.target.checked });
+                  }}
+                  color="primary"
+                />
+              }
+              label="Compact"
+            />
+            <FormControl className={classes.formControl}>
+              <Select
+                value={state.showTypes}
+                onChange={(event: React.ChangeEvent<{ value: unknown }>) => {
+                  setState({ showTypes: event.target.value as string });
+                }}
+                inputProps={{
+                  className: `${classes.select} ${classes.showTypesSelect}`
+                }}
+                MenuProps={{
+                  className: classes.showTypesMenu,
+                  transformOrigin: {
+                    vertical: 'bottom',
+                    horizontal: 'left'
+                  },
+                  getContentAnchorEl: null
+                }}
+              >
+                {
+                  Object.keys(assetsTypes).map(typeId =>
+                    (
+                      <MenuItem value={typeId} key={typeId}>
+                        <Radio
+                          checked={state.showTypes === typeId}
+                          color="primary"
+                        />
+                        {assetsTypes[typeId].label}
+                      </MenuItem>
+                    )
+                  )
+                }
+              </Select>
+            </FormControl>
+          </DialogFooter>
+          <EmbeddedLegacyEditors
+            showTabs={false}
+            dialogConfig={editDialogConfig}
+            setDialogConfig={setEditDialogConfig}
+          />
```

Comment:
Dialog manager

Reply:
This component is being moved to DialogManager by Hung